gateway/test/chains/ethereum/evm.nonce.test.ts

```diff
@@ -81,8 +84,10 @@ describe('unitiated EVMNodeService', () => {
 
 describe('EVMNodeService', () => {
   let nonceManager: EVMNonceManager;
+  const dbPath = '/tmp/evm-nonce2.test.level';
```

Comment:
You shouldn't expect `/tmp/` to always exist in UNIX systems. Also, the temporary files should be cleaned up after use.

Try `fs.mkdtemp()` and `fs.remove()` way as shown in `config-manager-v2.test.ts`. That ensures the generated paths are really yours and that they're cleaned up after use.